### DISCUSSION TITLE
Allow local users to set profile picture

### DISCUSSION THREAD

**Discussion by boultonn** [sct/overseerr#2245] [Feature Requests]
Plex users automagically pull their Plex profile photo in but currently local users (majority of my users) can't set a picture. Would be nice if they could.

**Comment by TheCatLady** [comment]
They already can. Local user avatars are pulled from Gravatar.

**Reply by bmpreston** [reply]
What is gravatar?  I just created the local user, in Overseerr.

**Comment by kybernetic-afk** [comment]
@TheCatLady How long does it take for an image to pull once setup in Gravatar?
